doc/ .browserlistrc

```diff
@@ -0,0 +1,4 @@
+# Browsers that we support
```

Comment:
Is the space in file name intentional?

Reply:
Unless I'm missing something, yes it is.

Reply:
@alextrepa I agree with Youssef: that's very odd.
![image](https://user-images.githubusercontent.com/4174207/134396518-f53e354f-8920-4763-b387-1ec21ab8fe5d.png)

Reply:
Yeah, you're right, it's fixed